images/build/php/7.4/Dockerfile

```diff
@@ -1,3 +1,3 @@
 # Build PHP 7.4
-FROM php-build-prereqs AS php74-build
+FROM oryxdevmcr.azurecr.io/private/oryx/php-build-prereqs AS php74-build
```

Comment:
Curious how do we push the images to the dev MCR or prod MCR with a `private`, `mirror` or `oss` tag, did we achieve it manually?

Reply:
If we'd like to update these basic layer images, such as a different version of go base image, we'll need to push a new one to the MCR, then update it there?

Reply:
Got it, thanks for explaining.